packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/modelparts/semantics/Initial.java

```diff
@@ -0,0 +1,19 @@
+package software.uncharted.terarium.hmiserver.models.dataservice.modelparts.semantics;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+import java.util.List;
+import software.uncharted.terarium.hmiserver.models.dataservice.modelparts.metadata.VariableStatement;
+
+@Data
+@Accessors(chain = true)
+public class Initial {
+	private String target;
+	private String expression;
+
+	@JsonAlias("expression_mathml")
+	private String expression_mathml;
```

Comment:
Why do you need an alias here?

Reply:
I didn't feel like cleaning up stuff until we have settled on the snake vs camel debate.